src/Keboola/StorageApi/Client.php

```diff
@@ -390,79 +390,123 @@ public function dropBucket($bucketId, $options = array())
         return $this->apiDelete($url);
     }
 
-    public function shareBucket($bucketId, $options = [])
+    public function shareBucket($bucketId, $options = [], $async = false)
```

Comment:
Tohle by mělo být součást options. Je to tak u jiných endpointů (např. workspace drop)